benchmark/lib/store.js

```diff
@@ -1,39 +1,142 @@
 import { logger } from '../../src/log/logger.js';
-import { S3Client, PutObjectCommand, GetObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocument, TransactWriteCommand } from '@aws-sdk/lib-dynamodb';
+import { randomUUID } from 'crypto';
```

Comment:
I made `srid()` which is a short random ID, it's shorter than UUID and for use it's random enough. it's in src/util.js